manifests/utils.pp

```diff
@@ -13,8 +13,8 @@
 #
 class snmpd::utils (
   $rsync_source  = "snmp_${::environment}/mibs",
-  $rsync_server  = hiera('rsync::server'),
-  $rsync_timeout = hiera('rsync::timeout','2')
+  $rsync_server  = lookup('::simp_options::rsync::server',  { 'default_value' => '127.0.0.1', 'value_type' => String}),
+  $rsync_timeout = lookup('::simp_options::rsync::timeout', { 'default_value' => '2', 'value_type' => String }),
```

Comment:
These should be static. The `lookup()` function does not look at the variable namespace, so if a user modifies `::simp_options` via profile and sets a different value in Hiera the values will diverge. The logic should go in the module's `<module_name>::data` function.

See: https://docs.puppet.com/puppet/latest/lookup_quick_module.html#more-about-the-function-data-provider